### ISSUE TITLE
Write script to convert `.nat` EUMETSAT files to intermediate file format

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/Satip#14]
## Detailed Description
Following on from #13 

## Features of this script

- [ ] Takes command-line arguments for directory of `.nat` files; and target directory for intermediate files
- [ ] Optionally reprojects (to be tracked in a separate issue)
- [ ] Optionally only saves a geographical subset of the data (perhaps with some handy human-readable shortcuts like "UK")
- [ ] The default behaviour should be that, when the script starts, it check through all the `.nat` files (recursively), and checks through all the already-converted files, and only converts data which is present in the `.nat` files but absent in the intermediate files.  Provide an option to 'clobber' existing files.
- [ ] Use all the CPU cores.

Related:

- [The ipython notebook I wrote in 2019 to convert and reproject `.nat` files to NetCDF](https://github.com/openclimatefix/satellite_image_processing/blob/master/notebooks/convert_native_to_reprojected_netcdf.ipynb)
- [Comparison I made of image file formats downloaded from EUMETSAT](https://github.com/openclimatefix/satellite_image_processing/blob/master/notebooks/compare_EUMETSAT_file_formats.ipynb)

**Comment by jacobbieker** [comment]
Yeah, I agree, keeping as much of the satellite specific stuff in Satip makes sense. I like generally keeping the packages as separate as possible.
